specification/resources/resource-manager/Microsoft.Resources/stable/2019-05-10/resources.json

```diff
@@ -1057,6 +1057,46 @@
         }
       }
     },
+    "/providers": {
+      "get": {
+        "tags": [
+          "Providers"
+        ],
+        "operationId": "Providers_ListAll",
+        "description": "Gets all resource providers for the tenant.",
+        "parameters": [
+          {
+            "name": "$top",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "description": "The number of results to return. If null is passed returns all providers."
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The properties to include in the results. For example, use &$expand=metadata in the query string to retrieve resource provider metadata. To include property aliases in response, use $expand=resourceTypes/aliases."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of resource providers.",
+            "schema": {
+              "$ref": "#/definitions/ProviderListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
     "/subscriptions/{subscriptionId}/providers/{resourceProviderNamespace}": {
```

Comment:
>"/subscriptions/{subscriptionId}/providers/{resourceProviderNamespace}" [](start = 4, length = 71)

You should add the equivalent path for tenant-level as well.

Reply:
Thanks for adding that.

---
In reply to: [304122974](https://github.com/Azure/azure-rest-api-specs/pull/6645#discussion_r304122974) [](ancestors = 304122974)

Reply:
Added.